NLog.Web/LayoutRenderers/AspNetItemValueLayoutRenderer.cs

```diff
@@ -56,18 +56,14 @@ public class AspNetItemValueLayoutRenderer : LayoutRenderer
         /// </summary>
         /// <param name="builder">The <see cref="StringBuilder"/> to append the rendered data to.</param>
         /// <param name="logEvent">Logging event.</param>
-        protected override void Append(StringBuilder builder, LogEventInfo logEvent)
+        protected override void DoAppend(StringBuilder builder, LogEventInfo logEvent)
```

Comment:
Maybe I'm missing something, but I dont see a backward-compatibility issue. Any subclass that calls base.Append will still experience the same behavior, the implementation changed but the behavior remains the same. If the subclass doesn't call base.Append then it is opting out of the base class behavior.
The DoAppend method is optional to override, but I would add to the docs that is the recommended way to write aspnet layout renderers that rely on the HttpContext since it makes the layout renderers more unit test friendly.